### ISSUE TITLE
Main navigation requires child roles

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#2997]
![image](https://github.com/swisspost/design-system/assets/1659006/5d5c2371-67f8-4680-abf1-b82643765a00)

A lighthouse test revealed that the main navigation child elements are missing a role to fulfill their purpose.

**Comment by alizedebray** [comment]
The `role="menu"` was removed from the `nav#post-internet-header-main-navigation` as part of the commit [58fb500](https://github.com/swisspost/design-system/commit/58fb500256d03fd114330d72dcd69e7c5bf7eee7#diff-7bb87cc5efc0674a0a9c45df430c83bc924edfa3d47c3554ef8c9d295f76d047). 
This error is therefore no longer reproducible.
